appcues/src/main/java/com/appcues/trait/appcues/TooltipTrait.kt

```diff
@@ -185,7 +191,7 @@ internal class TooltipTrait(
                                 contentSized.value = true
                             }
                             .styleShadowPath(style, tooltipPath, isSystemInDarkTheme())
-                            .clipToPath(tooltipPath)
+                            .clip(GenericShape { _, _ -> addPath(tooltipPath) })
```

Comment:
Ok, was that maybe to solve detekt ? in any case its ok cleanup

Reply:
yeah it was just to reduce function count for detekt, inlining a one line function that was just used in one spot